setup.cfg

```diff
@@ -127,18 +120,12 @@ install_requires =
     pluggy~=1.0
     psutil>=4.2.0, <6.0.0
     pygments>=2.0.1, <3.0
-    pyjwt<3
```

Comment:
What required pyjwt3 before?

Reply:
>pyjwt has been fixed upstream (flask-jwt-extended, via FAB), so we
don't need to require it ourselfs # Please enter the commit message
for your changes.